watson_developer_cloud/natural_language_understanding_v1.py

```diff
@@ -82,3 +82,24 @@ def analyze(self, features, text=None, url=None, html=None,
                             headers={'content-type': 'application/json'},
                             json=body,
                             accept_json=True)
+    def listModels(self):
+        """
+        Lists the custom models available for your service instance
+        :return: dict of available custom models
+        """
+        return self.request(method='GET', url='/v1/models',
+                            params={"version": self.version},
+                            accept_json=True)
+
+    def deleteModel(self, model_id=None):
```

Comment:
I was just mirroring how the features parameter is checked by the SDK in the analyze endpoint. Does the new commit fix things?